src/go/agwd/config/config.go

```diff
@@ -43,8 +43,8 @@ func LogLevel(l mconfig.AgwD_LogLevel) log.Level {
 }
 
 const (
-	ipv4Scheme = "ipv4"
-	ipv6Scheme = "ipv6"
+	ipv4Scheme = "tcp4"
```

Comment:
the naming here says the target should be `ipv4` or `ipv6`

https://github.com/grpc/grpc/blob/master/doc/naming.md

so  i guess we should map it to `tcp4` or `tcp6` when we open the listener?